routes/pictures.js

```diff
@@ -0,0 +1,84 @@
+import { keysToCamel } from '../common/utils';
+
+const express = require('express');
+const { db } = require('../server/db');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  try {
+    const pictures = await db.query('SELECT * FROM pictures');
+    res.status(200).json(keysToCamel(pictures));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+router.post('/', async (req, res) => {
+  try {
+    const { id, furnitureId, imageURL } = req.body;
+    const pictures = await db.query(
+      `INSERT INTO pictures (
+        id, furnitureId, imageURL,
+        )
+      VALUES (
+        $(id), $(furnitureId), $(imageURL)
+        )
+      RETURNING *;`,
+      {
+        id,
+        furnitureId,
+        imageURL,
+      },
+    );
+    res.status(200).json(keysToCamel(pictures));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+router.put('/:picturesId', async (req, res) => {
+  const { picturesID } = req.params;
```

Comment:
wrap this inside the try catch block so in the rare case that this doesn't work the proper error message is returned